src/main/resources/application.properties

```diff
@@ -1,8 +1,9 @@
 spring.application.name=includify
```

Comment:
We need not push these changes. You can keep it in your local. Once we move the db to cloud, then we can think of adding it here (although not in GitHub). Can u please revert this?